scripts/jobrunner.sh

```diff
@@ -53,6 +53,7 @@ EOF
 }
 
 cp jobrunner/bin/* /srv/jobrunner/bin/
+cp jobrunner/usr-local-bin/* /usr/local/bin
```

Comment:
Yeah, it would be nice to keep them together. But these end up owned by `jobrunner` and therefore running them as root is a priv escalation risk – or that's how it feels to me unless I'm missing something?